src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const isMarried = (personStatus) => {
+    if (!personStatus.isMarried) {
+      return 'I am not married';
+    }
+
+    return personStatus.sex === 'm'
+      ? `${personStatus.partnerName} is my wife`
+      : `${personStatus.partnerName} is my husband`;
+  };
+
+  return (
+    <section className="person">
+      <h2 className="person__name">
```

Comment:
```suggestion
    <section className="Person">
      <h2 className="Person__name">
```
Fix all cases to pass the tests
![image](https://user-images.githubusercontent.com/96409563/236748095-7073fd79-a5df-473a-b2ce-d1e45d063c2e.png)

Reply:
[witflash](https://github.com/witflash) [4 days ago](https://github.com/mate-academy/react_person/pull/1265#discussion_r1185156978)
There is no need to use CamelCase from CSS class names. You can name it just .person.
It's not a mistake but a general recommendation.

NadieinOleh reacted with thumbs up emoji
Author
@[NadieinOleh](https://github.com/NadieinOleh) NadieinOleh [3 days ago](https://github.com/mate-academy/react_person/pull/1265#discussion_r1185756151) • 
I didn` change CSS className, it was in the instance.